Source/Core/Core/Lua/LuaFunctions/LuaBitFunctions.cpp

```diff
@@ -0,0 +1,143 @@
+#include "LuaBitFunctions.h"
+#include "../LuaVersionResolver.h"
+#include "common/CommonTypes.h"
+
+namespace Lua
+{
+namespace LuaBit
+{
+class BitClass
+{
+public:
+  BitClass() {}
+};
+
+BitClass* bitInstance;
```

Comment:
seems like this should be static, since it isn't used outside the translation unit.